<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-DJQ94NE" name="EmpListExcelTest" tests="2" failures="1" timestamp="14 Nov 2022 15:14:41 GMT" time="21.572" errors="0">
  <testcase name="LoginTest" time="5.317" classname="com.OrangeHRMTestNG.OrangeHRM_EmployeeListintoExcel"/>
  <testcase name="pimEmplListIntoExcelTest" time="6.819" classname="com.OrangeHRMTestNG.OrangeHRM_EmployeeListintoExcel">
    <failure type="java.lang.IllegalArgumentException" message="The workbook already contains a sheet named &amp;apos;EmpListPage1&amp;apos;">
      <![CDATA[java.lang.IllegalArgumentException: The workbook already contains a sheet named 'EmpListPage1'
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.validateSheetName(XSSFWorkbook.java:870)
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.createSheet(XSSFWorkbook.java:827)
	at com.OrangeHRMTestNG.OrangeHRM_EmployeeListintoExcel.pimEmplListIntoExcelTest(OrangeHRM_EmployeeListintoExcel.java:68)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- pimEmplListIntoExcelTest -->
</testsuite> <!-- EmpListExcelTest -->
